foodplaner/users/models.py

```diff
@@ -1,3 +1,13 @@
+"""
+A model is the single, definitive source of information about your data. It contains the essential fields and behaviors of the data you’re storing. Generally, each model maps to a single database table.
+
+The basics:
+
+Each model is a Python class that subclasses django.db.models.Model.
+Each attribute of the model represents a database field.
+With all of this, Django gives you an automatically-generated database-access API
+"""
```

Comment:
Bitte unseren Spezialfall beschreiben und nicht Django allgemein. In der ersten Zeile darf nur ein Satz sein